docs/stories/stories.md

```diff
@@ -0,0 +1,79 @@
+# COGS User Stories
+
+## Roles:
+User
+Project owner
+Principal Investigator
+Project Authoriser
+Attribution Authoriser
+RSE
+System Administrator
+
+## Stories
+### Project creation
+- Users with valid, non-student institutional email addresses can create new projects as the project owner
```

Comment:
Can 'apply' for new projects yes.  Will re-word in current system.

Reply:
Sorry, "create" was simply easier phrasing for "create a database record". You'll notice I tend to use this kind of language a lot. Not a reflection of my preference of the wording in the UI. 😄